src/init.cpp

```diff
@@ -2000,6 +2000,13 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
         if (connect.size() != 1 || connect[0] != "0") {
             connOptions.m_specified_outgoing = connect;
         }
+        if (!connOptions.m_specified_outgoing.empty() && !connOptions.vSeedNodes.empty()) {
```

Comment:
This could have gone inside the condition right above.

Reply:
I guess @naumenkogs meant this:
```cpp
if (connect.size() != 1 || connect[0] != "0") {
    connOptions.m_specified_outgoing = connect;

    if (!connOptions.vSeedNodes.empty()) {
        LogPrintf("-seednode is ignored when -connect is used\n");
    }
}
```